core/src/main/scala/spinal/core/BaseType.scala

```diff
@@ -24,36 +24,32 @@ import spinal.core.internals._
 import scala.collection.Seq
 import scala.collection.mutable.ArrayBuffer
 
-trait TypeFactory{
+trait TypeFactory {
   def postTypeFactory[T <: Data](that: T) = that
 }
 
-/**
-  * Base type factory
+/** Base type factory
   */
 trait BaseTypeFactory extends BoolFactory with BitsFactory with UIntFactory with SIntFactory with VecFactory with SFixFactory with UFixFactory
 
-
-/**
-  * Base type Cast
+/** Base type Cast
   */
 trait BaseTypeCast extends SFixCast with UFixCast
 
-object BaseType{
-  final val isRegMask      = 1
+object BaseType {
+  final val isRegMask = 1
   final val isTypeNodeMask = 2
-  final val isVitalMask    = 4
-  final val isAnalogMask   = 8
-  final val isFrozen       = 16
+  final val isVitalMask = 4
+  final val isAnalogMask = 8
+  final val isFrozen = 16
```

Comment:
This is one thing that i fear the most, the formater has no artistic feelings about alignements.
That kind of patterns would be something to realy keep.

Reply:
This can be tuned with configuration: 

```
align.preset = more
align.multiline = true
```

Reply:
Magnificent, let's turn it on <3